journal-of-historical-linguistics.csl

```diff
@@ -111,15 +111,13 @@
           <text macro="title" prefix=" "/>
           <group prefix=".">
             <group>
-              <group suffix=".">
-                <names variable="editor translator">
-                  <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always" prefix=" In: " suffix=","/>
+              <group prefix=" " suffix=".">
+                <text term="in" text-case="capitalize-first" suffix=" "/>
+                <names variable="editor translator" delimiter="," suffix=",">
+                  <name delimiter=" " and="symbol" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
I'm not seeing initials in the bibliographies, so remove `initialize-with=""`. I also see names in regular order, so remove the `name-as-sort-order`. Finally, I'm not seeing a comma before the ampersand, so set `delimiter-precedes-last="never"`
I'm working off this example:
![image](https://cloud.githubusercontent.com/assets/128986/23823759/3abd04ce-0637-11e7-8499-883265744c76.png)
Might also be there are different ones? But this makes sense in the context of the style